### PR TITLE
Fixing master

### PR DESCRIPTION
**Pull request by danielkutik**
hi bjoern the last commit of yours is making some trouble :) still a little unstable after these fixes (timeouts etc)

### CODE REVIEW

**Comment by bkerler** [comment]
Yeah, I shouldn't add new stuff without testing it before :D I couldn't add your commit, as you changed the function label of info to check_info. It would have broken other code I use. Is there a specific reason why you wanted to rename it ? Regarding the loader db, I just pushed a commit, so thx :)

File: edl.py [hunk 1]
```diff
@@ -298,7 +298,7 @@ def run(self):
                 else:
                     print("Device is in EDL mode .. continuing.")
                     self.cdc.timeout = None
-                    sahara_info = self.sahara.info()
```
Comment (danielkutik) [inline edl.py:301]: if i execute this code, the `info()` method in `sahara.py` gets called... issue is there are kind of two of them. the one that i renamed and one for logging (`info("my logging msg with log level info")`) hence i renamed it

**Review by danielkutik — COMMENTED** [review]
Made a inline comment with the reason for renaming the method... Code did not execute for me without this change.

**Comment by bkerler** [comment]
Oh, yes, my fault. Guess I will have to review the last changes. Thx !

**Comment by bkerler** [comment]
Can you please try commit 5c9244b ? I've tested it and it should work fine now.
As for stability issues, I recommend the usb-cable used by the vendor.

**Comment by danielkutik** [comment]
if i reverted df35f3d520751fbebbf95a07ea969d01cf393494 i defenitly did not have the stability/timeout issues anymore so it was related to that.. I will retest masters later today or tomorrow and if i say any further issues i will investigate and reopen separate issue. closing this one for now. danke

**Pull request closed without merging**